content/community/user-stories.md

```diff
@@ -49,6 +49,7 @@ This page is intended as a place to collect user stories and feedback on Apache
 |[Huawei Cloud ROMA](https://www.huaweicloud.com/en-us/product/roma.html)|Apache camel powers the runtime engine for [Huawei Cloud ROMA](https://www.huaweicloud.com/en-us/product/roma.html) which is a commercial iPaaS offering by Huawei Cloud. ROMA integrates data, services, messages and devices under one unified platform which enables it's customers to hook up systems spanning across multiple Paas, Saas and cloud services.|
 |[ModusBox PortX](https://modusbox.com/portx-platform/)|PortX, the Integration Platform as a Service (IPaaS) for financial institutions, is built on the open source Apache Camel framework.|
 |[Guidewire Integration Framework](https://www.guidewire.com/blog/technology/cloud-integration-framework-the-right-tools-for-the-job/)| Integration Gateway is an open framework for developing integration apps that bridge Guidewire APIs with external apps and services. It is based on Apache Camel.| 
+|[Multi-Channel Framework (MCF)](https://www.felpfe.com/multi-channel-framework-mcf/)| Apache Camel Without Code! MCF is thoughtfully crafted to empower developers, granting them an intuitive framework for building, testing, and deploying intricate integration scenarios with unparalleled ease. By harnessing the powerful capabilities of Apache Camel, MCF remarkably simplifies the development workflow with its [configuration-based](https://www.felpfe.com/2023/06/30/multi-channel-framework-mcf-use-case-isx-sample-requirements/) approach, minimizing the complexity and amount of code required. This configuration-driven approach allows developers to achieve seamless integration using Camel with minimal coding effort| 
```

Comment:
Thanks for sharing your story with Camel and your use case with us! 

This is just my personal observation, I would probably adjust the wording a little bit because we need to be  neutral when sharing the word about tools, products and services that are based on Camel. I believe something along these lines should be fine:

```
MCF is uses Camel to empower developers, granting them an intuitive framework for building, testing, and deploying intricate integration scenarios without writing code. By harnessing the powerful capabilities of Apache Camel, MCF simplifies the development workflow with its [configuration-based](https://www.felpfe.com/2023/06/30/multi-channel-framework-mcf-use-case-isx-sample-requirements/) approach, minimizing the complexity and amount of code required. This configuration-driven approach allows developers to achieve seamless integration using Camel with minimal coding effort
```

Reply:
Got it.  I will update the  description.